src/cf-typography/src/cf-typography.less

```diff
@@ -13,6 +13,11 @@
 @webfont-medium: 'AvenirNextLTW01-Medium';
 @webfont-demi: 'AvenirNextLTW01-Demi';
 
+// Webfont variables
+@cf-fonts-path: '/fonts';
```

Comment:
Let's change this to `/static/fonts` to match the config Ross set up over in cfpb/cfgov-refresh#3569.

Reply:
Fixed